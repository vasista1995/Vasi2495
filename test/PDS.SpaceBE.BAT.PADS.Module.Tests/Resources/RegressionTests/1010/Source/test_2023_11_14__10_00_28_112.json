{
  "IdSource": "SPACEACT2:BAT:BE:VE344129B08:BE1I:FOL_Autovision_SmartS:A2PLWirePullTop:7265:236630618:4",
  "Data_flat_Metadata": {
    "Device": "L2891S",
    "ParameterClass": "PPAP",
    "CFComment": "FOL",
    "Segment": "A2PL",
    "FourDReport": "*",
    "ParameterName": "A2PLWirePullTop",
    "ParameterUnit": "g",
    "ParameterOper": "FOL_Autovision_SmartS",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "7265",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
    "CreatedTimestamp": "2023-10-07T08:09:22Z",
    "UpdatedTimestamp": "2023-11-14T15:37:58Z",
    "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
    "ExportedTimestampUtc": "2023-10-07T08:09:22Z",
    "ExportedTimestamp": "2023-10-07T08:09:22Z",
    "SampleTimestampUtc": "2023-10-07T01:09:22Z",
    "SampleTimestamp": "2023-10-07T08:09:22Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wire_PullForce/AL/50um/DS28/A2AL",
    "ChannelDescr": "A2PLWirePullTop\\DS20\\A2AL\\50 ",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/06/2015 00:41:33",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": "iscw8cc2082mwr",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 372677,
    "ViolCount": 1,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "4",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 8.0,
        "SpecTarget": 16.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "27,989875",
        "Median": "28,439",
        "Sigma": "1,3592052759399",
        "Min": "25,642",
        "Max": "29,947",
        "Q1": "27,213",
        "Q3": "28,6525"
      },
      "ControlLimits": {
        "ctrl_mean_low": 25.63,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 30.09,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 34.56,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 3.3,
        "ctrl_range_high": 7.18,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 29.485078125,
        "ExtEwmaMV": 28.7509203300076,
        "ExtMsMV": 0.639578083583231,
        "EwmaS": 1.25848564140575,
        "EwmaR": 3.66796229675434,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 236630618,
        "Seqnr": 1,
        "Value": 26.655,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 2,
        "Value": 28.313,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 3,
        "Value": 28.792,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 8,
        "Value": 29.947,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 5,
        "Value": 25.642,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 6,
        "Value": 27.399,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 7,
        "Value": 28.606,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      },
      {
        "SampleId": 236630618,
        "Seqnr": 4,
        "Value": 28.565,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN\t\nRC        : CHECK MACHINE HISTORY & CHEMICAL ANALYSIS\t\nCA        : NO FOUND ABNORMALITY\t\nPCDF    : NA\t\nPREE    : M TILMIZAN\t\nPC        : SENDRI"
      }
    ],
    "SampleSize": 8
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:FOL_Autovision_SmartS:A2PLWirePullTop:7265",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:VE344129B08:BE1I:FOL_Autovision_SmartS:A2PLWirePullTop:7265:236630618:4"
  }
}