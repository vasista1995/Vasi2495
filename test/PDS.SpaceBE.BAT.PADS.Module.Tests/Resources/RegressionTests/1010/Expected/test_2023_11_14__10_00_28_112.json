{
  "Data_flat_Metadata": {
    "ParameterOper": "FOL_Autovision_SmartS",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-07T01:09:22Z",
    "BeginTimestamp": "2023-10-07T08:09:22Z",
    "EndTimestampUtc": "2023-10-07T01:09:22Z",
    "EndTimestamp": "2023-10-07T08:09:22Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "L2891S",
    "ParameterClass": "PPAP",
    "Segment": "A2PL"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 1,
          "SampleId": 236630618,
          "Value": 26.655,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 2,
          "SampleId": 236630618,
          "Value": 28.313,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 3,
          "SampleId": 236630618,
          "Value": 28.792,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 8,
          "SampleId": 236630618,
          "Value": 29.947,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 5,
          "SampleId": 236630618,
          "Value": 25.642,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 6,
          "SampleId": 236630618,
          "Value": 27.399,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 7,
          "SampleId": 236630618,
          "Value": 28.606,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        },
        {
          "Seqnr": 4,
          "SampleId": 236630618,
          "Value": 28.565,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-07T08:09:22Z",
          "SampleTimestampUtc": "2023-10-07T01:09:22Z",
          "CreatedTimestamp": "2023-10-07T08:09:22Z",
          "CreatedTimestampUtc": "2023-10-07T01:09:22Z",
          "UpdatedTimestamp": "2023-11-14T15:37:58Z",
          "UpdatedTimestampUtc": "2023-11-14T08:37:58Z",
          "NumViolations": 1,
          "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "ViolationList": "8",
          "PrimaryViolation": "8",
          "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
          "SampleMean": 27.989875,
          "SampleMin": 25.642,
          "SampleMax": 29.947,
          "SampleMedian": 28.439,
          "SampleStdev": 1.3592052759399,
          "SampleQ1": 27.213,
          "SampleQ3": 28.6525,
          "SampleSize": 8
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "4",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 8.0,
          "target": 16.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 25.63,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 30.09,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 34.56,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 3.3,
          "ctrl_range_high": 7.18,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 8,
        "base_count": 8,
        "flagged_count": 0,
        "mean": 27.989875,
        "median": 28.439,
        "stdev": 1.2714201152156592,
        "range": 4.305,
        "min": 25.642,
        "max": 29.947,
        "q02": 25.78382,
        "q05": 25.99655,
        "q25": 27.213,
        "q75": 28.652500000000003,
        "q95": 29.542749999999998,
        "q98": 29.7853,
        "NumViolations": 1,
        "ViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
        "ViolationList": "8",
        "PrimaryViolation": "8",
        "PrimaryViolationComments": "FD        : A2PL WIREPULL 6 POINT RUN DOWN  RC        : CHECK MACHINE HISTORY   CHEMICAL ANALYSIS  CA        : NO FOUND ABNORMALITY  PCDF    : NA  PREE    : M TILMIZAN  PC        : SENDRI",
        "Samples": "236630618"
      },
      "ParameterName": "A2PLWirePullTop",
      "ParameterUnit": "g",
      "ChannelId": "7265",
      "SourceDataLevel": "C",
      "channel_name": "Wire_PullForce AL 50um DS28 A2AL",
      "channel_description": "A2PLWirePullTop DS20 A2AL 50 ",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-07T01:09:22Z",
      "created_timestamp": "2023-10-07T08:09:22Z",
      "updated_timestamp_utc": "2023-11-14T08:37:58Z",
      "updated_timestamp": "2023-11-14T15:37:58Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T08:37:58Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:FOL_Autovision_SmartS:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:VE344129B08:BE1I:FOL_Autovision_SmartS:A2PLWirePullTop:7265:236630618:4",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-07T08:09:22Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:FOL_Autovision_SmartS::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}